app/models/clickdata.py

```diff
@@ -11,19 +11,14 @@
 
 # batch get has a 100 item limit
 # https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchGetItem.html
-def chunks(index, n=100):
+def _chunks(index, n=100):
```

Comment:
Renamed according to Python convention because these functions are intended to be kept private.